.devcontainer/devcontainer.json

```diff
@@ -3,6 +3,9 @@
   "dockerFile": "Dockerfile",
   "extensions": ["ms-vscode.cpptools"],
   "runArgs": ["--security-opt", "label=disable"],
-  "containerEnv": {"WORKSPACE_DIR": "${containerWorkspaceFolder}"}
+  "containerEnv": {"WORKSPACE_DIR": "${containerWorkspaceFolder}"},
+  "settings": { 
+		"terminal.integrated.shell.linux": "/bin/bash"
```

Comment:
If you happen to have VS Code settings sync turned on, and you normally use a shell other than `bash` (I use `fish`), then codespaces will complain about that shell not being present and fail to open a terminal. Somehow this works fine when doing containerized local development, but not in codespaces.